hw/i386/virt/virt.c

```diff
@@ -226,6 +226,8 @@ static void virt_machine_state_init(MachineState *machine)
             
     fw_cfg = fw_cfg_init(machine, smp_cpus, mc->possible_cpu_arch_ids(machine), vms->apic_id_limit);
     fw_cfg_add_i16(fw_cfg, FW_CFG_MACHINE_ID, X86_VIRT);
+    fw_cfg_add_i64(fw_cfg, FW_CFG_PCIEXBAR_BASE, PCI_HOST_PCIEXBAR_BASE);
```

Comment:
@liujing2 this is one way to do it. Another way, which may be better would be to actually create a file. That will allow us to keep this logic generic and add more fields in the future, one for each segment.